### PR TITLE
Adding Speakers to the Feed.

### LINKED ISSUE #8: Enable folders with in UI

### LINKED ISSUE #20: Import the Speakers

### PR DESCRIPTION
**Pull request by punkcoder**
- Added functionality to 'switch email' boxes
- Worked on moving the helper scripts to helperscripts/
- generateemail.py now compiles all of the data from email, speakers, and sponsors

### CODE REVIEW

**Comment by kernelmethod** [comment]
Looks good for the most part. I think we should try to add Amie and the keynote before we update the site with this though.

**Comment by kernelmethod** [comment]
Also -- I should've done this a while ago, but I'm going to look into whether there's an easier way to generate these templates. I just committed a small change to Kurt's bio (literally just making his name uppercase) and it, uh, was not super fun. :upside_down_face:

**Comment by punkcoder** [comment]
@kernelmethod all of that Information is being populated directly from sessionize, we need to get Amie into Sessionize and get Kurt's bio updated. When we do that I can rebuild them quickly.

**Comment by punkcoder** [comment]
@kernelmethod from src run ./helperscripts/generatespeakerhtml.py and you should be able to generate the speakers, then running ./helperscript/generateemail.py will rebuild the email boxes.

File: src/helperscripts/generatespeakerhtml.py [hunk 1]
```diff
@@ -27,6 +27,12 @@
 for speaker_pre in speakers:
     speaker = speaker_pre
     speaker['description'] = sessionLookup[speaker['sessions'][0]['id']]
+
+    #srsly hate hacks like this
```
Comment (punkcoder) [inline src/helperscripts/generatespeakerhtml.py]: @kernelmethod this was the fastest way to fix this permanently....

**Review by punkcoder — COMMENTED** [review]
Okay at this point we should be good to go... if I can get the approve thumbs up then it will go live.

File: src/helperscripts/generatespeakerhtml.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    template = Template(data)
+
+for speaker_pre in speakers:
+    speaker = speaker_pre
+    speaker['description'] = sessionLookup[speaker['sessions'][0]['id']]
+
+    # Capitalize name if necessary
+    name_components = speaker['fullName'].split(" ")
+    speaker['fullName'] = " ".join(c.title() for c in name_components)
+    print(speaker['fullName'])
```
Comment (kernelmethod) [inline src/helperscripts/generatespeakerhtml.py:34]: Is this any better? Only slightly less hacky than your original workaround I'm afraid

**Review by kernelmethod — APPROVED** [review]

**Pull request merged**